src/LocalizationWrapper.jsx

```diff
@@ -1,33 +1,36 @@
-import React, {Component, PropTypes} from 'react';
+import React, { Component, PropTypes } from 'react';
 import get from 'lodash.get';
 import defaultLocalizer from './util/localize-formatter';
 
 export default (ComposedComponent, messages = {}, customLocalizer) => {
   class ConfigureLocalization extends Component {
-    constructor(props) {
-      super(props);
+    static localize(key, values) {
+      const message = get(messages, key, null);
```

Comment:
Every app I've used with i18n has a flat messages bundle, so based on that experience, and given how slow get is, I wouldn't use it unless here at NF our bundles explicitly are nested.